lib/evm_database.rb

```diff
@@ -32,7 +32,11 @@ def self.seedable_model_class_names
   end
 
   def self.seed_primordial
-    seed(PRIMORDIAL_CLASSES)
+    if ENV['SKIP_PRIMORDIAL_SEED'] == 'true' && MiqDatabase.count > 0
```

Comment:
Agreed, I end up putting True, tru, truee... :laughing: 

Just setting the env variable is enough.